Sources/MapboxNavigation/OrnamentsController.swift

```diff
@@ -207,7 +238,7 @@ extension NavigationMapView {
                 
                 streetLabelLayer.sourceLayer = sourceLayerIdentifier
                 streetLabelLayer.lineOpacity = .constant(1.0)
-                streetLabelLayer.lineWidth = .constant(20.0)
+                streetLabelLayer.lineWidth = .expression(Expression.routeLineWidthExpression())
```

Comment:
If I’m not mistaken, #155 intentionally made this line layer a rather wide fixed width to reduce the changes that the query would come back with no results. It is possible for the query to return irrelevant results (cross streets etc.), but we would filter out those irrelevant results by comparing to the electronic horizon road name data.

Reply:
Thanks for the suggestion, I'll remove the road width change.